### PR TITLE
chore(submodule): upgrade brise

### PR DESCRIPTION
**Pull request by Prcuvu**
This will require `bash` (included in Git for Windows) to build `brise` data. Weasel expansion pack no longer exists.

### CODE REVIEW

File: build.bat [hunk 1]
```diff
@@ -146,15 +146,15 @@ exit /b
 
 :build_data
 rem call :build_essay
+if not defined BASH_DIR (
+  set BASH_DIR=C:\Program Files\Git\usr\bin
```
Comment (lotem) [inline build.bat]: How about adding the path to `DEVTOOLS_PATH`?
The variable is already appended to `PATH` at the beginning of `build.bat`, and can be defined by user in `env.bat` for multiple dev tools. We don't need to set `PATH` twice or change `build.bat` every time we need something new.
Guessing the default path doesn't help when user do not have Git bash installed, or installed elsewhere.
In that case just let `cmd` complain `bash` not found.
Like for other tools, we can add its default install path to `env.bat.template` for convenience.

File: build.bat [hunk 2]
```diff
@@ -146,15 +146,15 @@ exit /b
...
 copy %WEASEL_ROOT%\README.md output\README.txt
-copy %WEASEL_ROOT%\brise\essay.txt output\data\
-copy %WEASEL_ROOT%\brise\default.yaml output\data\
-copy %WEASEL_ROOT%\brise\symbols.yaml output\data\
-copy %WEASEL_ROOT%\brise\preset\*.yaml output\data\
-copy %WEASEL_ROOT%\brise\supplement\*.yaml output\data\
-if not exist output\expansion mkdir output\expansion
-copy %WEASEL_ROOT%\brise\extra\*.yaml output\expansion\
+cd %WEASEL_ROOT%\brise
+bash -c "export brise_dir=. rime_dir=../output/data && bash rime-install"
```
Comment (lotem) [inline build.bat]: Optional: `rime-install` can be called from outside `brise` directory, so we can omit the `cd` commands before and after.

**Comment by lotem** [comment]
Maybe file another issue: shall we preload *less* input schemas?
`bash rime-install :preset` include prevailing input methods in every major Chinese speaking regions, and should be good enough for most people.

`librime>=1.2.10` preserves the user copy of `*.yaml` if it's not found in shared data. So it won't cause problems for user of removed preload schemas. (need testing)

File: build.bat (hunk 1 above)
Comment (Prcuvu) [reply build.bat]: This is not based on the modified `build.bat` in `build` branch, I can not add it here.
I will definitely do that when the new `build.bat` is done.

**Comment by Prcuvu** [comment]
Then are we going to include those other than preset schemata in expansion pack? Then I will have to do it all over again.

**Comment by lotem** [comment]
I prefer to deprecate expansion set.
People seldom need multiple schemas from the expansion installer. Better install individual package on demand.
There were no good means to install them back then; it was a work-around. Today, we have a rime-install script. Not ready for everyone, but it is being improved. 

(To be continued...)

**Comment by lotem** [comment]
Let me explain my classification of schemas in `brise`.
There used to be 3 directories:
* `preset`: schemas that are meant to instantly useable; bundled, installed and provide prebuilt binaries;
* `supplement`: bundled and installed source files, user has to build binaries after enabling; because these are less often needed and take a long time to build all;
* `extra`: the "rest" of the collection are seldom needed; do not bundle to save installer size. They are here only because it's the only place to go before every input method has their own repo - the split.

Now `brise` continue to serve as a collection of input schemas, but only listing repo names, not including source files. With a package manager, bundled with the IME distribution is not the only user friendly way to install schemas; it even becomes a disadvantaged method - longer update cycle and wasting bandwidth on unused data.
That's why the `supplement` category is no longer installed, in my plan. Let user download and install their schema of choice on demand.

There is a make target `minimal` which creates a pruned dataset, even smaller than `preset`. Used for small installer that could conveniently do online updates.
However this change is a challenge for upgrading IME clients especially rime data packages for Linux distros.

**Comment by Prcuvu** [comment]
I think I get the point. Although there should be a GUI package manager for pulling other schemata, we can just leave it till later.

**Comment by lotem** [comment]
Shall we
 - make `preset` the default make target in brise? (I'm working on a commit)
 - delete weasel expansion installer?

**Comment by Prcuvu** [comment]
> make preset the default make target in brise? (I'm working on a commit)

Agreed.

> delete weasel expansion installer?

I have done this since it is no longer being used.

**Comment by lotem** [comment]
Pushed my commits to `brise`. Please sync submodule and feel free to merge.

**Pull request merged**